oak-run/src/test/java/org/apache/jackrabbit/oak/run/DataStoreCommandTest.java

```diff
@@ -158,7 +159,12 @@ public void setup() throws Exception {
     }
 
     private static Data prepareData(StoreFixture storeFixture, DataStoreFixture blobFixture, int numBlobs,
-        int numMaxDeletions, int missingDataStore) throws Exception {
+                                    int numMaxDeletions, int missingDataStore) throws Exception{
+        return prepareData(storeFixture, blobFixture, numBlobs, numMaxDeletions, missingDataStore, false);
+    }
+
+    private static Data prepareData(StoreFixture storeFixture, DataStoreFixture blobFixture, int numBlobs,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'prepareData(StoreFixture, DataStoreFixture, int, int, int, boolean)' has an NPath complexity of 3240](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5079914)